README.md

```diff
@@ -55,8 +56,41 @@ To increase logging level to include info and debug messages, set the environmen
 `DEBUG`.
 
 ## Usage
+```
+git clone https://github.com/metno/discovery-metadata-catalog-ingestor
+
+cd discovery-metadata-catalog-ingestor
+
+mkdir workdir
+
+cd workdir
+
+mkdir rejected
+
+```
+First initialize the config.yaml or use environment-variables.
+
+```
+dmci:
+  distributors:
+    - file
+    - pycsw
+  distributor_cache: workdir
+  rejected_jobs_path: workdir/rejected
+  max_permitted_size: 100000
+  mmd_xsl_path: path/to/'mmd-to-geonorge.xsl'
+  mmd_xsd_path: path/to/'mmd_strict.xsd'
+  path_to_parent_list: path/to/parent-uuid-list.xml
```

Comment:
Should we provide an example file for this? or atleast describe what this file is in the readme

Reply:
parent-uuid-list.xml that is

Reply:
is that just about parent-uuid-list.xml or should we need write more about the xsl and xsd  files also?

Reply:
good point, maybe add something about the mmd-repo: https://github.com/metno/mmd for those